examples/cluster/gcp/main.tf

```diff
@@ -24,9 +24,18 @@ resource "redpanda_cluster" "test" {
   throughput_tier   = var.throughput_tier
   zones             = var.zones
   allow_deletion    = true
-  tags = {
-    "key" = "value"
-  }
+  #   tags = {
+  #     "key" = "value"
+  #   }
+  #   gcp_private_service_connect = {
+  #     enabled               = true
+  #     global_access_enabled = true
+  #     consumer_accept_list = [
+  #       {
+  #         source = "projects/123456789012"
+  #       }
+  #     ]
+  #   }
```

Comment:
Is this meant to be commented on for reference on how to enable PSC? if so, can you put a small message saying so above the commented code?

Reply:
Partly. It's also commented out because we can't run integration tests against it